src/main/kotlin/no/nav/familie/pdf/pdf/PdfUtils.kt

```diff
@@ -155,23 +155,43 @@ object PdfUtils {
                     setDestination(navigeringDestinasjon)
                 },
             )
-            when (element["visningsVariant"].toString()) {
-                VisningsVariant.TABELL_BARN.toString() -> {
-                    val listeMedAlleBarn = lagListeMedAlleBarn(element["verdiliste"] as List<*>)
-                    listeMedAlleBarn.forEachIndexed { index, barn ->
-                        val barneIndeksTekst = "Barn " + (index + 1).toString()
-                        add(lagTabell(barn, barneIndeksTekst))
-                    }
-                }
-                VisningsVariant.VEDLEGG.toString() -> {
-                    håndterVedlegg(element["verdiliste"] as List<*>, this)
+            val verdiliste = element["verdiliste"] as List<*>
+
+            if ("visningsVariant" in element) {
+                håndterVisningsvariant(element["visningsVariant"].toString(), verdiliste, this)
+            } else {
+                håndterRekursivVerdiliste(verdiliste, this)
+            }
+            add(LineSeparator(SolidLine()))
+        }
+
+    private fun håndterVisningsvariant(
+        visningsVariant: String,
+        verdiliste: List<*>,
+        seksjon: Div,
+    ) {
+        when (visningsVariant) {
+            VisningsVariant.TABELL_BARN.toString() -> {
+                val listeMedAlleBarn = lagListeMedAlleElementer(verdiliste, "Navn")
+                listeMedAlleBarn.forEachIndexed { index, barn ->
+                    val barneIndeksTekst = "Barn " + (index + 1).toString()
```

Comment:
Generelt sett: 
Tekster som skal være i pdfen bør helst ikke hardkodes da disse er ulike på norsk og engelsk. Vi har "hardkodet" en variant av dette i `familie-ef-soknad-api`, men da med mulighet for to språk:

https://github.com/navikt/familie-ef-soknad-api/blob/main/src/main/kotlin/no/nav/familie/ef/s%C3%B8knad/utils/Spr%C3%A5ktekster.kt